test/framework/src/main/java/org/elasticsearch/indices/analysis/AnalysisFactoryTestCase.java

```diff
@@ -77,7 +77,6 @@ private static String toCamelCase(String s) {
         .put("edgengram", MovedToAnalysisCommon.class)
         .put("keyword", MovedToAnalysisCommon.class)
         .put("letter", MovedToAnalysisCommon.class)
-        .put("lowercase", MovedToAnalysisCommon.class)
```

Comment:
We still need this ? The tokenizer is deprecated, not yet removed ?

Reply:
Again, this is checking for lucene classes that no longer exist.